### ISSUE TITLE
Integration with ApiComparer

### ISSUE THREAD

**Issue by andreasohlund** [fsprojects/Paket#1012]
Crazy idea :)

We're [contemplating](https://github.com/ParticularLabs/APIComparer/issues/46) an api for [ApiComparer](https://github.com/ParticularLabs/APIComparer). Does paket have any integration points where it would make sense to list api changes, warn about breaking changes etc?

`paket outdated` with a potential `--show-diffs` looks interesting. I guess it could be just a generated link eg. `http://apicomparer.particular.net/compare/rabbitmq.client/3.4.3...3.5.0` or some deeper integration where semver compliance is checked and updates with breaking changes are flagged in the output of the command in some way.

Thoughts?

**Comment by forki** [comment]
Yes I guess that could make sense. Opening a website would be a valid way
to do this.
